docs/source/support/release-notes.rst

```diff
@@ -6,6 +6,70 @@ Release notes
 
 This page contains release notes for the SDK.
 
+.. _release-0-13-28:
+
+0.13.28 - 2019-10-04
+--------------------
+
+JSON API - Experimental
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- Returning archived and active/created contracts from ``/command/exercise``
+- Returning archived and active contracts from ``/command/exercise``
```

Comment:
There seems to be duplication the first two bullets here

Reply:
good catch, merged